### ISSUE TITLE
add polyfill for `node:events.once`

### ISSUE THREAD

**Issue by birdlavv** [unjs/unenv#54]
Reproduction steps
1. Visit https://stackblitz.com/edit/github-gvgyln-ersphr?file=server%2Fapi%2Ftest.ts,nuxt.config.ts
2. Run `npm run build`

The reproduction was made using
  1. a fresh Nuxt 3 project
  2. using the `jose` package on a server-side api route
  3. using the `vercel-edge` nitro preset
  
causes the following error:
```
1: import * as http from 'http';
2: import * as https from 'https';
3: import { once } from 'events';
            ^
4: import { JOSEError, JWKSTimeout } from '../util/errors.js';
5: import { concat, decoder } from '../lib/buffer_utils.js';


 ERROR  'once' is not exported by node_modules/unenv/runtime/node/events/index.mjs, imported by node_modules/jose/dist/node/esm/runtime/fetch_jwks.js
```

**Comment by Dreaming-Codes** [comment]
Any workaround?

**Comment by officialpiyush** [comment]
+1, any updates?

**Comment by officialpiyush** [comment]
currently `vuefire` module needs the `firebase-admin` package to provide SSR features. firebase-admin module internally uses `jose`, due to which the builds fail at build time, it would be great to know any updates on this, or any possible workarounds.

**Comment by pi0** [comment]
PR more than welcome 🙏🏼

**Comment by DanSnow** [comment]
Here is a workaround with Nuxt:
1. install `events`
2. add the following to your `nuxt.config.ts`

```ts
export default defineNuxtConfig({
  hooks: {
    'nitro:config': (nitroConfig) => {
      nitroConfig.unenv = {
        alias: {
          events: require.resolve('events/'),
        },
      }
    },
  },
})

```

**Comment by officialpiyush** [comment]
> PR more than welcome 🙏🏼

@pi0 opened a PR with the fix #68
